islandora.module

```diff
@@ -2215,3 +2215,12 @@ function islandora_islandora_breadcrumbs_backends() {
     ),
   );
 }
+
+/**
+ * Implements hook_islandora_solution_pack_child_relationships
+ */
+function islandora_islandora_solution_pack_child_relationships($cmodels) {
+  // Return empty arrays from core implementation so that these keys always
+  // exist when the hook is called, even if no module responds.
+  return array('predicate' => array(), 'prefix' => array());
```

Comment:
Should this not return the `fedora-rels-ext:isMemberOf` and `fedora-rels-ext:isMemberOfCollection` relationships like in the `islandora.api.php` (above)? 

`islandora:isPageOf` might be better for `islandora_paged_content`, but I am open to debate that.

Reply:
Hey Jared. I tried to implement these in the solution packs that use the relationships. So there are 4 PRs that define the relationships: 

https://github.com/Islandora/islandora_solution_pack_collection/pull/208
https://github.com/Islandora/islandora_solution_pack_book/pull/203
https://github.com/Islandora/islandora_solution_pack_compound/pull/147
https://github.com/Islandora/islandora_solution_pack_newspaper/pull/161

Reply:
Does this make sense Jared? Just want to make sure I addressed your comments here.

Reply:
Ping @whikloj

Reply:
Sorry...totally slipped off my plate. I'm fine with this.

Reply:
Ok, so we are all good? Nice. I just saw a new button: "Resolve Conversartion", should i try it out? Thanks @jonathangreen for waiting and @whikloj for coming back to this!